src/stream_management/mod_stream_management_sasl2.erl

```diff
@@ -28,7 +28,7 @@ hooks(HostType) ->
 -spec sasl2_stream_features(Acc, #{c2s_data := mongoose_c2s:data()}, gen_hook:extra()) ->
     {ok, Acc} when Acc :: [exml:element()].
 sasl2_stream_features(Acc, _, _) ->
-    Resume = #xmlel{name = <<"resume">>, attrs = [{<<"xmlns">>, ?NS_STREAM_MGNT_3}]},
+    Resume = #xmlel{name = <<"sm">>, attrs = [{<<"xmlns">>, ?NS_STREAM_MGNT_3}]},
```

Comment:
Where is this change described in the XEP? It seems to be against https://xmpp.org/extensions/xep-0388.html#example-2

Reply:
That's a bug in the XEP that I found testing with Conversations. Conversations and Prosody already implemented using <sm/> and teh XSF group discussed changing the XEP to use such implementation instead, see https://github.com/xsf/xeps/pull/1215/files#r1321210647